src/Protocol/Protocol_1_14.cpp

```diff
@@ -733,6 +720,7 @@ UInt8 cProtocol_1_14::GetProtocolEntityType(const cEntity & a_Entity) const
 				case PType::pkFirework: return 26;
 				case PType::pkWitherSkull: return 92;
 			}
+			break;
```

Comment:
Shouldn't the ```break;``` statement be inside the ```{ }```

Reply:
Yeah - My bad. The GitHub web interface made it seem like the break was somewhere else.